odysseus/odysseus_tree/board/tpu/plugin_user_property_timestamp.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/time.h>
+
+
+#include "nanomq/plugin.h"
+
+/*
+ * How to compile:
+ * gcc -I../ -fPIC -shared plugin_user_property.c -o plugin_user_property.so
+ */
+
+
+int cb(void *data)
+{
+	char **property = data;
```

Comment:
i personally prefer char *property[]

Reply:
I [edited this example from nanomq](https://github.com/nanomq/nanomq/blob/master/nanomq/plugin/plugin_user_property.c) where they use that.  I can change it tho.

Reply:
Eh I like `**`, I call strings `char* string` anyways, so its just a pointer to a string